app/components/UserProfile/useProfile.ts

```
@@ -0,0 +1,199 @@
+        dispatch({ type: 'SET_POINTS', payload: Number(userPoints) });
+        dispatch({ type: 'SET_BALANCE', payload: userPoints.toString() });
+      } catch (error) {
+        console.error('Error fetching points:', error);
+        toast.error('Failed to fetch points');
+        dispatch({ type: 'SET_BALANCE', payload: '0' });
+      }
+    };
+    fetchPoints();
+  }, [account]);
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: publicClient.readContract

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)